go.mod

```diff
@@ -1,9 +1,9 @@
-module github.com/GetStream/stream-chat-go/v2
+module github.com/GetStream/stream-chat-go/v3
 
 go 1.12
 
 require (
 	github.com/getstream/easyjson v0.0.0-20190923162548-5adc7b1f33fa
 	github.com/pascaldekloe/jwt v1.5.0
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
```

Comment:
while updating, maybe for https://pkg.go.dev/github.com/stretchr/testify@v1.5.1?tab=doc